service-python/pdfgenerator/src/lib/helper_functions.py

```diff
@@ -5,7 +5,12 @@
 from dateutil.relativedelta import relativedelta
 
 
-def pdf_helper_0000(data):
+def pdf_helper_asthma(data):
```

Comment:
I moved this code from pfd_helper_all because I believe it is only needed for the shared medications template which is used in `asthma-v1` template.
Line 43:
`        {{medicine.authoredOn.strftime('%m/%d/%Y')}} at {{medicine.authoredOn.strftime('%H:%M %p %Z')}}
`
The above line reformat the date object, right?.

Reply:
Ah yes you're correct. Disregard

Reply:
Maybe it is needed for 7101 as well?

```
VroV1Tests > positive7101Case01() FAILED
    org.springframework.web.client.HttpServerErrorException$InternalServerError: 500 : "{"claimSubmissionId":"7001","status":"ERROR","reason":"'str object' has no attribute 'strftime'"}"
```

Reply:
I moved it because if authoredOn is empty it may fail, for now I am moving it back because that is not the problem at hand